src/common/Throttle.h

```diff
@@ -332,25 +332,38 @@ class OrderedThrottle {
   uint32_t waiters = 0;
 };
 
+struct Bucket {
```

Comment:
This was discussed with Jason, as we only have one user of Bucket, just keep it in TokenBucketThrottle. Don't move it outside currently.